config/initializers/sentry.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# This file was previously config/initializers/raven.rb
+
+require 'sentry/event_scrubber'
+
+# This custom Transport class is need to log the error
+# This needs to be manually tested
```

Comment:
Is there a ticket for manually testing this? I just want to make sure the validation doesn't get forgotten! :)

Reply:
I didn't make a ticket for it. It was just a note to myself that I forgot to removed